solver/src/driver.rs

```diff
@@ -162,14 +162,21 @@ impl Driver {
 
         self.metrics.liquidity_fetched(&liquidity);
 
+        let gas_price = self
+            .gas_price_estimator
+            .estimate()
+            .await
+            .context("failed to estimate gas price")?;
+        tracing::debug!("solving with gas price of {}", gas_price);
```

Comment:
Should we use a default price instead?

Reply:
I think it's correct to fail in that case. We definitely need the gas price estimation to work for submitting our transactions.